### PR TITLE
docs(datepicker): add demo for outsideClick option

### PR DESCRIPTION
**Pull request by ghost**
add demo for outsideClick option(the datepicker should close when user click outside of the datepicker's field)
related #3636

### CODE REVIEW

File: demo/src/app/components/+datepicker/demos/outside-click/outside-click.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 col-md-4 form-group">
+    <input class="form-control" placeholder="Datepicker" bsDatepicker [outsideClick]="true">
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/outside-click/outside-click.html]: Add some text that will let a user know what differences these two datepickers have.

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 2]
```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Outside click',
+        anchor: 'outside-click',
+        component: require('!!raw-loader?lang=typescript!./demos/outside-click/outside-click.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/outside-click/outside-click.html'),
+        description: `<p>You can close a datepicker out a field using outsideClick property</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: Datepicker closes after outside click by default. To change this behavior, use `outsideClick` property.

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

File: demo/src/app/components/+datepicker/demos/outside-click/outside-click.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<div class="row">
+  <div class="col-xs-12 col-12 col-sm-6 col-md-4 form-group">
+    <label>You can close datepicker by clicking outside the datepicker's field</label>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/outside-click/outside-click.html]: сhange this to `[outsideClick]="true"`

File: demo/src/app/components/+datepicker/demos/outside-click/outside-click.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+    <input class="form-control" placeholder="Datepicker" bsDatepicker [outsideClick]="true">
+  </div>
+  <div class="col-xs-12 col-12 col-sm-6 col-md-4 form-group">
+    <label>You can't close datepicker by clicking outside the datepicker's field</label>
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/demos/outside-click/outside-click.html]: `[outsideClick]="false"`

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Needs to fix conflicting files (update your branch from latest development and solve conflicts)

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Got this merge conflict after updating branch
mergeconflict

**Comment by ghost** [comment]
update and testing my branch according to Yevheniia comments(conflicts in files) which I can't see

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Review by valorkin — APPROVED** [review]

**Pull request merged**